mock_server/package-lock.json

```diff
@@ -1,8 +1,96 @@
 {
   "name": "mock_server",
   "version": "1.0.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
Were the changes to package-lock.json necessary? If not, maybe we can revert it to the master version to clean up the PR a bit.